templates/member/admin/memberip.html

```diff
@@ -37,4 +37,24 @@
             {% endfor %}
         </ul>
     </div>
+
+    {% if ":" in ip %}
+    <p>
+        {% blocktrans %}
+            Liste des membres dont la dernière IP connue fait partie du bloc {{ network_ip }}
```

Comment:
Suggestion, à voir ce que tu en penses.

```suggestion
            Liste des membres dont la dernière IP connue fait partie du bloc <code>{{ network_ip }}</code>
```

Si accepté, il faudrait faire de même un peu plus haut, mais je ne peux mettre de suggestion car c'est hors du diff de cette PR.

Reply:
Faut que je teste le rendu sur mon zds local. Tu as checké comment ça rendait ?

Reply:
Oui : 

![Rendu avec les IPs en chasse fixe](https://user-images.githubusercontent.com/1417570/119004110-8f8fe980-b98e-11eb-91b8-bc124247b338.png)

Reply:
J'ai fait le changement. L'adresse se détache un peu. C'est pas forcément utile en soit, je ne sais pas bien ce que ça apporte de pouvoir lire une IP, surtout une IPV6. Mais ça ne fait pas de mal non plus.